ReText/config.py

```diff
@@ -33,7 +33,7 @@
     QSpinBox,
     QTabWidget,
     QVBoxLayout,
-    QWidget,
+    QWidget, QGroupBox, QHBoxLayout,
```

Comment:
Please use one import per line and keep the list sorted (you can use `isort` tool for that).